README.md

```diff
@@ -84,13 +84,24 @@ $ cabal build curiosity
 $ ./dist-newstyle/build/x86_64-linux/ghc-8.6.5/curiosity-0.1.0.0/x/curiosity/build/curiosity/cty serve
 ```
 
-And finally, we can build a binary with Nix:
+We can build a binary with Nix:
 
 ```
 $ nix-build -A curiosity
 $ ./result/bin/cty serve
 ```
 
+And finally, we can run a local dev environment containing curiosity and a Nginx reverse proxy using:
+
+```
+$ nix-build -A integration-tests.interactiveIntegrationEnv
+$ ./result/bin/run-interactive-integration-env
+$ # Or alternatively from a nix-shell via an injected alias
+$ nix-shell
```

Comment:
There is a specific shell available with `nix-shell default.nix -A shell`. Within that shell, you have access to a built `cty` binary. Maybe in that shell, instead of providing a script that actually calls `nix-build` (that's what your alias does if I follow correctly), we could provide directly the `run-interactive-integration-env` (i.e. it would be built as part of the shell, instead of being built when you call the alias, and the alias would no longer be needed).

And maybe that `run-interactive-integration-env` could rely on the `cty` that is already available in the shell, instead of crafting a `$PATH` as part of `runCuriosity`.

Actually, that shell is the local environment I was talking about above. So it makes sense for me that it provides `cty`, the man pages (already done), a script to run Nginx, and the `run-interactive-integration-env` (that could be renamed to `run-procfile` or `run`) to run everything together.

Reply:
It definitely makes sense. It actually aligns with something else I wanted to do: add a new procfile flavor using a locally-built (ie. a incremental build) cty.

It's a little bit more of work though: it'd need to add a second profile flavor using a locally-built cty instead of something coming from the store.

I'll probably need to generalize this procfile "flavor" concept to support the integration tests anyways.

Let's address this is a subsequent PR.